aiohttp_csrf/__init__.py

```diff
@@ -0,0 +1,118 @@
+from functools import wraps
+
+from aiohttp import web
+
+
+from .policy import AbstractPolicy
+from .storage import AbstractStorage
+
+
+__version__ = '0.0.1'
+
+REQUEST_STORAGE_KEY = 'aiohttp_csrf_storage'
+DECORATOR_FIELD_NAME = 'aiohttp_csrf_check'
+
+
+async def get_token(request):
+    storage = request.get(REQUEST_STORAGE_KEY)
+
+    if storage is None:
+        raise RuntimeError(
+            "Install aiohttp_csrf middleware in your aiohttp.web.Application",
+        )
+
+    return await storage.get(request)
+
+
+async def generate_token(request):
+    storage = request.get(REQUEST_STORAGE_KEY)
+
+    if storage is None:
+        raise RuntimeError(
+            "Install aiohttp_csrf middleware in your aiohttp.web.Application",
+        )
+
+    return await storage.generate_new_token(request)
+
+
+def check(handler):
+    handler.__dict__[DECORATOR_FIELD_NAME] = True
```

Comment:
Can we just copy paste https://github.com/django/django/blob/master/django/views/decorators/csrf.py#L49 ?